Add clarification for accessible name for details / summary

scottaohara opened the pull request:
Closes #212

@stevefaulkner @zcorpan feedback on the ordering of the updated/added bullets?


<!--
    This comment and the below content is programatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/html-aam/pull/215.html" title="Last updated on Jun 28, 2019, 3:34 AM UTC (3167677)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/html-aam/215/e8e3d68...3167677.html" title="Last updated on Jun 28, 2019, 3:34 AM UTC (3167677)">Diff</a>

scottaohara committed:
change code to markdown

scottaohara committed:
update details/summary accName comp

additional updates to convert to markdown.

zcorpan commented on index.html:
```diff
@@ -5826,31 +5829,34 @@ <h4>Other Form Elements Accessible Description Computation</h4>
-            If the <code>summary</code> element has an <a href="https://www.w3.org/TR/wai-aria-1.1/#aria-label"><code>aria-label</code></a> or an <a href="https://www.w3.org/TR/wai-aria-1.1/#aria-labelledby"><code>aria-labelledby</code></a> attribute the <a class="termref">accessible name</a> is to be calculated using the algorithm defined in <a href="" class="accname">Accessible Name and Descrip ...
+            If the `summary` element has an <a href="https://www.w3.org/TR/wai-aria-1.1/#aria-label">`aria-label`</a> or an <a href="https://www.w3.org/TR/wai-aria-1.1/#aria-labelledby">`aria-labelledby`</a> attribute the <a class="termref">accessible name</a> is to be calculated using the algorithm defined in <a href="" class="accname">Accessible Name and Description: Computation and API Mapping ...
+          </li>
+          <li>Otherwise use `summary` element subtree.</li>
```
While you're here, this could say which `summary` element. That is, the first such child, if there is one.

scottaohara replied:
Will give this a check when I get back to my computer

scottaohara committed:
clarify which summary

the accName algorithm applies to the first summary which is a direct child of the details element.
